lib/openid/fetchers.rb

```diff
@@ -1,5 +1,4 @@
 require 'net/http'
-require 'openid'
```

Comment:
I think you'll need to `require 'openid/version'` instead, as `OpenID::VERSION` is referenced further down in this file.